warehouse/templates/pages/help.html

```diff
@@ -153,9 +153,9 @@ <h3 id="mirroring">{{ mirroring() }}</h3>
           If you need to run your own mirror of PyPI, the <a href="https://pypi.org/project/bandersnatch/">bandersnatch project</a> is the recommended solution. Note that the storage requirements for a PyPI mirror would exceed 1 terabyte—and growing!
         </p>
 
-        <h3 id="notifications">{{ notifications() }}</h3>
+        <h3 id="project-release-notifications">{{ project_release_notifications() }}</h3>
         <p>
-          While PyPI itself does not offer a way to get notified when new releases are uploaded, there are several third-party services that offer comprehensive monitoring and notifications for package releases and vulnerabilities such as <a href="https://pyup.io" target="_blank" rel="noopener">pyup.io</a> and <a href="https://libraries.io/" target="_blank" rel="noopener">libraries.io</a>.
+          PyPI itself does not offer a way to get notified when a project uploads new releases. However, here are several third-party services that offer comprehensive monitoring and notifications for project releases and vulnerabilities such as <a href="https://pyup.io" target="_blank" rel="noopener">pyup.io</a> and <a href="https://libraries.io/" target="_blank" rel="noopener">libraries.io</a>.
```

Comment:
Nice rewording! But "here" should be "there".

Reply:
Done.